.gitignore

```diff
@@ -36,3 +36,7 @@ yarn-debug.log*
 
 # Ignore IDE files
 .idea/
+
+# Environment variables
+.env.development
+.env.test
```

Comment:
Actually we don't need `env.development` and `env.test` files. You should create `env.example` file. And use it as example for necessary ENV variables.

Reply:
Got your thought, resolved)